global.json

```diff
@@ -0,0 +1,5 @@
+{
+  "sdk": {
+    "version": "3.0.100"
```

Comment:
I'm not a big fan of global.json files as they are too constraining on the dev environment. Generally prefer docker as a means to enforce reproducible builds.

Reply:
removed it